module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,16 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+#include <array>
+#include <iostream>
+
+void removeVowels (std::vector<std::string>& fullText) {
+
+const std::array<char,12> vowelArr {'a', 'A', 'e', 'E', 'i', 'I', 'o', 'O', 'u', 'U', 'y', 'Y'};
+
+    for (auto& element : fullText) {
+        for (const auto& vowel : vowelArr) {
```

Comment:
const auto without reference because becase reference is 4 or 8 byte size and char have only 1 byte size
```suggestion
        for (const auto vowel : vowelArr) {
```

Reply:
Agree. Since you don't modify variable(const), you can pass variable that already exists instead of creating copy of it.